cmd/openshift/openshift.go

```diff
@@ -25,6 +25,9 @@ import (
 
 func main() {
 	logs.InitLogs()
+	// Valid values are DEBUG, INFO, WARN, ERROR, FATAL, and PANIC.
+	// Empty string results in a no-op.
+	serviceability.InitLogrus(os.Getenv("OPENSHIFT_LOGRUS_LEVEL"))
```

Comment:
It seems like a user would expect that setting `-v=5` (or some value; 2, 3?) would result in getting debug logs for all components.  Having them set "I want verbose logging" and also set "I really want verbose logging" seems pretty frustrating.

Reply:
@smarterclayton @deads2k can I get a ruling here?  Should this be set via an env var (I think that's what @smarterclayton proposed) or just piggy back on the `--loglevel` setting?

Reply:
@smarterclayton @deads2k bump

Reply:
@smarterclayton @deads2k last bump before this goes in as an env var.....